students/itsabhi/Lesson3/mailroom.py

```diff
@@ -0,0 +1,73 @@
+#! /usr/bin/env python3
+
+
+def sending_thanks(d):
+    donor_hist = d
+    unique_donors = []
+    for entry in donor_hist:
+        name = entry[0]
+        if name not in unique_donors:
+            unique_donors.append(name)
+
+    donor = input("Type the Full Name of the donor here: ")
+    if donor == "list":
+        print(unique_donors)
+        donor = input("Type the Full Name of the donor here: ")
+    if donor not in unique_donors:
+        donor_hist.append([donor])
+        amount = input("Enter Donation value: ")
+        donor_hist[-1].append([amount])
+#        print(donor_hist)
+
+    else:                       #donor appears in list of donors
+        for entry in donor_hist:
+            if donor == entry[0]:
+                amount = input("Enter Donation value: ")
+                entry[1].append(int(amount))
+                continue
+    #       print(donor_hist)
+    print("Hello {}, Just wanted to drop a note of thanks for your recent donation of ${}.".format(donor, amount))
+    return 1
+
+
+def second_integer(list):
+    return list[1]
```

Comment:
It's a good idea to avoid using Python built-ins like `list` as variable names. One convention is to use a trailing underscore like `list_`, or something specific to your program like `donation_list`